tooling/minitest/src/tests/compute_align.rs

```diff
@@ -0,0 +1,198 @@
+use crate::*;
+
+/// Helper which builds code to assert that compute_align with given type gives the expected align.
```

Comment:
The comment should say that this is for *sized* types only. (same in `compute_size.rs`)